### PR TITLE
Adding notification post GitHubPat input

### PR DESCRIPTION
**Pull request by kanika1894**
Since the user isn't aware of the repo analysis call being made post the input of GitHubPat, it is likely that he will try creating a new flow using Ctrl+Shift+P which creates faulty telemetry logs. Therefore, adding additional notification to prevent such scenarios.

### CODE REVIEW

**Comment by bishal-pdMSFT** [comment]
@kanika1894 we already had notification for repo analysis. Has that regressed?

**Comment by kanika1894** [comment]
> @kanika1894 we already had notification for repo analysis. Has that regressed?

No, that is still there
In configure class, there is a ``` getSelectedPipeline() ``` function which has that notification - Analyzing your repo
But that comes quite late in the flow, and till there is nothing for user display.
The time difference is more specially when the gitHubPat entered is incorrect.

**Comment by bishal-pdMSFT** [comment]
okay got it. so that notification is for local analysis.

I think we should change that notification to say "fetching templates"

File: src/configure/resources/messages.ts [hunk 1]
```diff
@@ -108,4 +108,5 @@ export class Messages {
     public static cannotCreateGitHubRepository = "New GitHub repository could not be created as the repository with the same name already exists";
     public static languageNotSupported = "The language of the repository selected is not supported.";
     public static UnableToGetTemplateParameters = "Unable to get parameters for the selected template.";
+    public static FetchingRepoDetails = "Fetching repository details";
```
Comment (bishal-pdMSFT) [inline src/configure/resources/messages.ts:111]: String could be `Analyzing your repository`

Comment (kanika1894) [reply src/configure/resources/messages.ts]: updated

**Pull request merged**